zion/src/main/scala/edu/uci/ics/cloudberry/zion/common/Config.scala

```diff
@@ -31,5 +34,36 @@ object Config {
     FiniteDuration(split(0).toLong, split(1))
   }
 
+  object MemorySize extends Enumeration {
+    val KB: Int = 1024
+    val MB: Int = 1024*1024
+    val GB: Int = 1024*1024*1024
+  }
+
+  def parseFrameLengthLimit(memoryString: String): Int = {
+    val size = memoryString.substring(memoryString.length-2)
```

Comment:
I tested it by changing the following value in `.conf` file: `"8MB"` `"8 MB"` `"8 TB"` `"8 GB"` `"2GB"` `"1KB"`, the results are expected. How should I present the test for this case?